vault/dendron.topic.images.md

```diff
@@ -2,35 +2,42 @@
 id: a91fd8da-6895-49fe-8164-a17acd8d9a17
 title: Images
 desc: ''
-updated: 1659026813451
+updated: 1662866892933
 created: 1595562035825
-stub: false
 ---
-    
-## Copy & Paste
+## Summary
 
-Dendron automatically inserts images in your clipboard using the `> Paste Image` command. The image is saved to `${currentFileDir}/assets` by default. This feature requires the [Paste Image](https://link.dendron.so/vscode-paste-image) VS Code extension.
+Dendron extends regular Markdown image support by allowing custom CSS properties. Images can also be pasted into a file from your clipboard.
+
+## Use Cases
+
+* add images and screenshots to notes
+* modify images using inline YAML map
+
+## Features
+### Copy & Paste
+
+Dendron automatically inserts images in your clipboard using the `> Paste Image` command. The image is saved to `${currentFileDir}/assets/images` by default. This feature requires the [Paste Image](https://link.dendron.so/vscode-paste-image) VS Code extension.
 
 <a href="https://www.loom.com/share/e1f6d207a1134f42b7a1a7750658acec">
 <img style="" src="https://cdn.loom.com/sessions/thumbnails/e1f6d207a1134f42b7a1a7750658acec-with-play.gif"> </a>
 </a>
 
-## Inserting Saved Images
+### Inserting Saved Images
 
 An easy way to insert an image which is saved locally to your Markdown note is to do the following:
 
-- Copy / save the image in your `assets/images` folder located in your vault,
-- In Dendron navigate to the note where you wish to input the image(s), and type:
-- `![Image_Description](assets/images/File_Name.png)`
-  - Be sure to replace `Image_Description` and `File_Name.png` with the description of the image and the actual filename
+1. Copy / save the image in your `assets/images` folder located in your vault,
+2. In Dendron navigate to the note where you wish to input the image(s), and type:
+3. `![Image_Description](assets/images/File_Name.png)`. Be sure to replace `Image_Description` and `File_Name.png` with the description of the image and the actual filename
 
-## Extended Images
+### Extended Images
 
 Dendron allows you to extend your images by adding custom CSS properties to
 them. This can allow you to display an image at a specific size, make it float
 (text wrapping around image), add a border, or much more. Extended images work both in [[dendron://dendron.dendron-site/dendron.topic.preview]] and in [[Publishing|dendron.topic.publish]].
 
-### Syntax
+#### Syntax
```

Comment:
would move to getting started

Reply:
I was thinking a good approach might be to keep the examples of extended images in the Features and move the syntax to the Getting Started. That would keep the separation between jobs-to-be-done and step-by-step instructions. Let me know your thoughts.

Reply:
sounds good